panels/site_sections/panel_app_management.py

```diff
@@ -0,0 +1,7 @@
+from panels import *
+
+
+@all_renderable(c.STUFF)
+class Root:
+    def index(self, session):
+        return {'apps': session.query(PanelApplication).order_by('applied').all()}
```

Comment:
As I mentioned, we'll need to eventually add some admin code to allow us to actually manage the applications.